opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/stixCyberObservable-test.js

```diff
@@ -138,6 +146,73 @@ describe('StixCyberObservable resolver standard behavior', () => {
     expect(stixCyberObservable.data.stixCyberObservableAdd.observable_value).toEqual('8090');
     networkTrafficInternalId = stixCyberObservable.data.stixCyberObservableAdd.id;
   });
+  it('should stixCyberObservable SSH_key created/update', async () => {
+    // Create the stixCyberObservable
+    const STIX_OBSERVABLE_TO_CREATE = {
+      type: 'SSH-Key',
+      SSHKey: {
+        key_type: 'rsa',
+        public_key: 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGmZ9d3b0QYpU2c9m7xKJ5V2rQy4s1aZr7Jk8Qw0t6u9',
+        fingerprint_sha256: 'a35f9c12e84b07d46ab13e95c728f06d2a8e41bb9d630cfa7419e2568b30d96f'
+      },
+    };
+    const stixCyberObservableSSH = await queryAsAdmin({
```

Comment:
```suggestion
    const stixCyberObservableSSH = await queryAsAdminWithSuccess({
```
there is some common "success check" in this method